atp/client.go

```diff
@@ -99,18 +108,19 @@ type executionEntry struct {
 
 type client struct {
 	atpVersion                       int64
-	channel                          ClientChannel
+	rawChannels                      ClientChannel
```

Comment:
After much thought, I can kinda see the name `rawChannels` , and I suppose just the plural is an improvement over `channel`, but it's a bit ... raw. More specifically, this captures the interface to the ATP signal input and output channels, right? So maybe `atpChannels` would make the semantics a bit more clear here?

Reply:
Changed as discussed in scrum to rawAtpChannels.